### PR TITLE
feat(acc): partner space doc

### PR DESCRIPTION
**Pull request by jsudraud**
Added a new category: Partners
+ Sub-category: Partner Space

This documentation explains how to use the upcoming Partner Space. In the future, this category will also include general information about Partners (e.g., how to become a Scaleway Partner). 

Do not merge until the Partner Center is released

### CODE REVIEW

File: partners/partner-space/how-to/edit-client-information.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+meta:
+  title: How to edit client information
+  description: This page explains how to edit client information the Partner Space.
```
Comment (RoRoJ) [inline partners/partner-space/how-to/edit-client-information.mdx:4]:
```suggestion
  description: This page explains how to edit client information from the Partner Space.
```

File: partners/partner-space/how-to/edit-client-information.mdx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+categories:
+  - partner-space
+---
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- Scaleway Partner status
+
+1. Click the **See more** button (<Icon name="more" />) to the right of the client’s creation date.
```
Comment (RoRoJ) [inline partners/partner-space/how-to/edit-client-information.mdx:21]: I think there should be a step to say how to get to this (the client's creation date)

File: partners/partner-space/how-to/edit-client-information.mdx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  validation: 2024-05-27
+  posted: 2024-05-27
+categories:
+  - partner-space
+---
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- Scaleway Partner status
```
Comment (RoRoJ) [inline partners/partner-space/how-to/edit-client-information.mdx:19]:
```suggestion
- Scaleway Partner status
- [Added a client](/partners/partner-space/how-to/add-client/)
```

File: partners/partner-space/how-to/lock-client-organization.mdx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  paragraph: This page explains how to lock a client's Organization from the Partner Space.
+tags: partner-space 
+dates:
+  validation: 2024-05-27
+  posted: 2024-05-27
+categories:
+  - partner-space
+---
+
+If you are confronted with a client who does not respect Scaleway’s Terms of Service, you can lock their Organisation.
```
Comment (RoRoJ) [inline partners/partner-space/how-to/lock-client-organization.mdx:16]:
```suggestion
If you are confronted with a client who does not respect Scaleway’s Terms of Service, you can lock their Organization. 
```

File: partners/partner-space/how-to/edit-client-information.mdx (hunk 1 above)
Comment (nerda-codes) [inline partners/partner-space/how-to/edit-client-information.mdx:4]:
```suggestion
  description: This page explains how to edit client information from the Partner Space.
```

File: partners/partner-space/how-to/lock-client-organization.mdx [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+categories:
+  - partner-space
+---
+
+If you are confronted with a client who does not respect Scaleway’s Terms of Service, you can lock their Organisation. 
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- Scaleway Partner status
```
Comment (RoRoJ) [inline partners/partner-space/how-to/lock-client-organization.mdx:21]:
```suggestion
- Scaleway Partner status
- [Added a client](/partners/partner-space/how-to/add-client/)
```

File: partners/partner-space/how-to/lock-client-organization.mdx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+  <Message type="important">
+     Locking an Organization prevents its owner to order or manage resources. .
```
Comment (RoRoJ) [inline partners/partner-space/how-to/lock-client-organization.mdx:24]:
```suggestion
     Locking an Organization prevents its owner from ordering or managing resources.
```

File: partners/partner-space/how-to/lock-client-organization.mdx [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    </Message>
+
+1. Click the **See more** button (<Icon name="more" />) to the right of the client’s creation date.
```
Comment (RoRoJ) [inline partners/partner-space/how-to/lock-client-organization.mdx:27]: As before

File: partners/partner-space/how-to/unlock-client-organization.mdx [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+  validation: 2024-05-27
+  posted: 2024-05-27
+categories:
+  - partner-space
+---
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- Scaleway Partner status
```
Comment (RoRoJ) [inline partners/partner-space/how-to/unlock-client-organization.mdx:19]:
```suggestion
- Scaleway Partner status
- [Added a client](/partners/partner-space/how-to/add-client/)
```

File: partners/partner-space/index.mdx [hunk 9]
```diff
@@ -0,0 +1,41 @@
+---
+meta:
+  title: Partner Space Documentation
+  description: Dive into Scaleway Partner Space with our quickstart guide, how-tos, and more.
+---
+
+<ProductHeader 
+   productName="Partner Space"
+   productLogo="iam"
+   description="As a Scaleway Partner, you benefit from a dedicated interface within the console: the Partner Space. It empowers you to manage your client efficiently, from creating their Organizations to monitoring their consumption for accurate billing."
```
Comment (RoRoJ) [inline partners/partner-space/index.mdx:10]:
```suggestion
   description="As a Scaleway Partner, you benefit from a dedicated interface within the console: the Partner Space. It empowers you to manage your clients efficiently, from creating their Organizations to monitoring their consumption for accurate billing."
```

File: partners/partner-space/how-to/lock-client-organization.mdx (hunk 7 above)
Comment (ldecarvalho-doc) [reply partners/partner-space/how-to/lock-client-organization.mdx:27]:
```suggestion
1. Click <Icon name="more"> to the right of the client’s creation date. 
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Comment by ldecarvalho-doc** [comment]
congrats on your first doc <3

File: partners/partner-space/how-to/lock-client-organization.mdx (hunk 7 above)
Comment (nerda-codes) [inline partners/partner-space/how-to/lock-client-organization.mdx:27]:
```suggestion
1. Click <Icon name="more" /> to the right of the client’s creation date. 
```

**Review by jcirinosclwy — APPROVED** [review]

**Pull request merged**